.github/workflows/test-addons.yml

```
@@ -0,0 +1,28 @@
+name: Test Addons
+
+on:
+  push:
+    branches: [ "main" ]
+    paths-ignore:
+      - '**.md'
+      
+jobs:
+  test-addons:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: start-minikube
+        uses: konveyor/tackle2-operator/.github/actions/start-minikube@main
+      - name: Build image in minikube
+        run: |
+          export SHELL=/bin/bash
+          eval $(minikube -p minikube docker-env)
+          make build-image
+      - name: install-tackle
+        uses: konveyor/tackle2-operator/.github/actions/install-tackle@main
+        with:
+          tackle-hub-image: tackle2-hub:latest
+          tackle-image-pull-policy: IfNotPresent
+
+      - name: test windup addon
+        uses: mundra-ankur/tackle2-addon-windup/.github/actions/test-e2e-windup@main
```

make sure we don't forget

```suggestion
        uses: konveyor/tackle2-addon-windup/.github/actions/test-e2e-windup@main
```